solr.cfg

```diff
@@ -39,12 +39,13 @@ java_opts =
   -server
   -Xms${settings:solr-min-ram}
   -Xmx${settings:solr-max-ram}
+
 index =
-    name:allowedRolesAndUsers   type:string stored:false multivalued:true
```

Comment:
@tisto 
According to the wiki page on http://wiki.apache.org/solr/Atomic_Updates:

>The core functionality of atomically updating a document requires that all fields in your SchemaXml must be configured as stored="true" except for fields which are <copyField/> destinations -- which must be configured as stored="false". This is because the atomic updates are applied to the document represented by the existing stored field values.

So this a solr requirement.